### PR TITLE
updated readme addressing reviewer comments

### PR DESCRIPTION
**Pull request by naqvia**
Addressing the following reviewer comments:

> 8. I also have some suggestions on GitHub: 1) In README.md, add software dependencies (for example, VEP, ANNOVAR, InterVar, AutoPVS1, vt, Bcftools); add where to get variant_summary.txt.gz and submission_summary.txt.gz before using them in select-clinVar-submissions.R, otherwise users will confuse when reading “Custom workflow example run”, until find what are they in FAQ; 2) In User-Guide, it would be great to add code of running AutoPVS1. It’s okay not to add if it is too complicated.

### CODE REVIEW

**Comment by naqvia** [comment]
@jungkim2 Can you add the code for running AutoPVS1 in the User-guide?

File: README.md [hunk 1]
```diff
@@ -43,7 +49,7 @@ AutoGVP Requirements (recommended to place all in the `data/` folder):
 
 ### Custom workflow example run
 1. [Prepare input files](https://github.com/diskin-lab-chop/AutoGVP/wiki/User-Guide#custom-input-workflow---step-by-step) by running VEP, ANNOVAR, InterVar, and AutoPVS1.
-2. Run `select-clinVar-submissions.R` with desired conflict resolution parameter ("latest", "most_severe") and optional ClinGen Concept ID list.
+2. Download [database files](https://github.com/diskin-lab-chop/AutoGVP/blob/main/scripts/download_db_files.sh) and run `select-clinVar-submissions.R` with desired conflict resolution parameter ("latest", "most_severe") and optional ClinGen Concept ID list.
```
Comment (jungkim2) [inline README.md:52]: isn't conflict resoluation parameter only required when ClinGen ConceptID list is provided?

File: README.md [hunk 2]
```diff
@@ -43,7 +49,7 @@ AutoGVP Requirements (recommended to place all in the `data/` folder):
```
Comment (jungkim2) [inline README.md:49]: for ClinVar VCF, it is now an option for CAVATICA workflow right? 
should we be explicit about that? 

required for custom workflow and optional for CATATICA?

**Review by jungkim2 — CHANGES_REQUESTED** [review]
I have modified WIKI user guide adding code to run AutoPVS1.

**Review by jungkim2 — APPROVED** [review]

**Pull request merged**